src/misc/mntent.c

```diff
@@ -20,6 +20,34 @@ int endmntent(FILE *f)
 	return 1;
 }
 
+static inline int decode1(char** in_buf, char** out_buf,  const char* from, const char to) {
+	if(strncmp(from, *in_buf, strlen(from)) == 0) {
```

Comment:
Thanks for having a look at the code.

No, the length isn't always 4.

I chose a really obvious and pedestrian style, like not putting in the 'obvious' 4 here, because I am afraid of making exactly those kinds of mistakes.

Reply:
Checking with Godbolt: on clang with `-O2` all the `strlen` here get replaced with constants.  No need for us to do the compilers job.

Reply:
Hmm, `memcmp` gets more inlined than `strncmp`, probably because the latter depends on locale.  Changing that now!